src/components/bs5/navbar/navbar.scss

```diff
@@ -191,6 +191,11 @@
                 }
             }
 
+            &:focus {
```

Comment:
We just need to add higher z-index to navbar__header.
At the moment the bottom outline is cropped
![image](https://github.com/user-attachments/assets/a2ec3053-90de-4de8-b905-3e969db5346f)

Reply:
z-index has been added to fix the outline.